runner/entrypoint.sh

```diff
@@ -44,7 +44,7 @@ if [ ! -d /runner ]; then
   exit 1
 fi
 
-sudo chown -R runner:docker /runner
+chown -R runner:docker /runner
```

Comment:
Have you tested this? Are you able to attach a screenshot to the PR showing this still works without sudo?

Reply:
Let me know how to test this and I will do so

Reply:
It would be great if you could try deploying actions-runner-controller with the modified chart and the runner image and see if a simple GitHub Actions workflow runs through!

FYI, to build a custom runner image, you can run:

```
NAME=$DOCKER_USER/actions-runner-controller \
  make docker-build docker-push
```

Also https://github.com/summerwind/actions-runner-controller#developing